### ISSUE TITLE
/friendica url is a 404

### ISSUE THREAD

**Issue by strk** [friendica/friendica#2209]
There's an 'Information' link in the menu accessible at the top-right corner of Vier theme.
The second entry ("Information") points to /friendica, which is a 404 here.
The other two entries ("Help" and "Guide") work fine.
This is as of version 3.4.3.

**Comment by annando** [comment]
This really should work. You can check with other servers.

**Comment by tobiasd** [comment]
is mod/friendica.php present on your system? this is the file to generate /friendica

**Comment by fabrixxm** [comment]
Please check also you don't have any "friendica*" file in your root friendica folder.

**Comment by strk** [comment]
I do have mod/friendica.php but also a friendica.log and a
friendica_test_data.sql file in the root friendica folder.

Removing "friendica.log" indeed fixes the issue, even if
"friendica_test_data.sql" remains there.

Note that "friendica.log" was created by the "Log" admin panel,
specifying "friendica.log" as the log file.

**Comment by strk** [comment]
Version is 3.5-dev (now that /friendica works I can tell for sure :)

**Comment by fabrixxm** [comment]
yep. I had same problem with "friendica.log" file and /friendica returning 404. I need to investigate why...

**Comment by tobiasd** [comment]
I just ```touch'ed friendica.log``` and could not reproduce this ;-)

**Comment by strk** [comment]
Just tried, as of commit 44afff9514fcf9a194eb7120779ceedbf4eae347,
and touching friendica.log takes me from working to a non-working
(shift-reload on firefox, to avoid caches) information page.

**Comment by fabrixxm** [comment]
could be an issue with modrewrite config? I had this issue on an apache based install, but I cannot reproduce on my lighttpd

**Comment by strk** [comment]
Could be. It's on Apache here, with rewrite directives:

```
  RewriteCond %{REQUEST_FILENAME} !-f
  RewriteCond %{REQUEST_FILENAME} !-d
  RewriteRule ^(.*)$ index.php?pagename=$1 [E=REMOTE_USER:%{HTTP:Authorization},L,QSA]
```

**Comment by tobiasd** [comment]
easy way out could be to rename the default name for the log file

**Comment by annando** [comment]
@strk BTW: You **never ever** should place logfiles in the web root. The logged data could reveal information about your system that could be used by attackers.

**Comment by tobiasd** [comment]
I thought that is why .log and .out are not served by default with the shipped .htaccess

**Comment by strk** [comment]
I think the admin page prompting you for a log file should let you
know what should or should not be done, and avoid creating problems.

**Comment by redmatrix** [comment]
This can happen if using Apache with the MultiViews option. MultiViews is evil - I always get rid of it, but I believe it is set by default in all Debian-based distros. Basically if you visit a path /foo and there is any file at all in the root folder with that basename - foo.gif or foo.txt or foo.bash - Apache tries to serve it up without even opening your index.php

**Comment by guzzisti** [comment]
I came across this issue as well. After renaming the "friendica.log" to something else everything is working fine.
Currently my instance runs shared hosting with apache, so i have limited options to influence the apache config.

Friendica ships with a default log settings of "friendica.log" which will - under specific circumstances - render the /friendica route broken, so i suggest to change the default setting to something else.

**Comment by MrPetovan** [comment]
Thanks for the warning, I believe we can do something about it.

**Comment by MrPetovan** [comment]
Fixed by #7291
